cmd/scw/testdata/test-all-usage-baremetal-server-list-usage.stderr.golden

```diff
@@ -0,0 +1,24 @@
+List all created servers.
+
+USAGE:
+  scw [global-flags] baremetal server list [flags] [arg=value ...]
+
+EXAMPLES:
+  List all servers on your default zone
+    scw baremetal server list
+
+ARGS:
+  [order-by]          Order of the servers (created_at_asc | created_at_desc)
+  [tags.{index}]      Filter servers by tags
```

Comment:
are we sure about this ? 

can we add a test to demonstrate this use case ?

Reply:
I've added a test for it. It works